model_compression_toolkit/core/common/mixed_precision/sensitivity_evaluation.py

```diff
@@ -89,10 +89,12 @@ def __init__(self,
                                                       fw_impl.count_node_for_mixed_precision_interest_points,
                                                       quant_config.num_interest_points_factor)
 
-        self.ips_distance_fns, self.ips_axis = self._init_metric_points_lists(self.interest_points)
+        self.ips_distance_fns, self.ips_axis = self._init_metric_points_lists(self.interest_points,
```

Comment:
add a comment here explaining the default behavior (that we run normalized MSE when not running hessian-based, you can also explain why in a sentence), otherwise we won't be able to figure out why we did it if we'll look at this one year from now